### DISCUSSION TITLE
Pages aren't loading right, and its telling me Javascript is Diabled..when its not

### DISCUSSION THREAD

**Discussion by Sp3llbound** [Alex313031/thorium#401] [Q&A]
as titles says...    no pages are loading anymore...    I tried reinstalling..  and it signed me out of everything..  and now its saying Javascript isnt enabled..  when it is...      I disabled experimentall features also....  nothing is working..

**Comment by gz83** [comment]
The problem you encountered cannot be reproduced. It is recommended to completely uninstall and reinstall the latest version of the browser.

**Reply by Sp3llbound** [reply]
ok...  thanks
